include/gdsync/core.h

```diff
@@ -57,6 +57,7 @@ enum gds_create_qp_flags {
 };
 
 typedef struct ibv_qp_init_attr_ex gds_qp_init_attr_t;
+typedef struct ibv_qp_attr gds_qp_attr_t;
```

Comment:
in mp_query_print_qp() (new function in my branch) I need the struct ibv_qp_attr to report QP info